tests/test_pds4_create_xml_index_blackbox.py

```diff
@@ -18,6 +18,12 @@
         'golden_file,new_file,cmd_line',
         [
             # Testing --limit-xpaths-file
+            # Executable command: pds4_create_xml_index ../test_files/labels
```

Comment:
I think the idea was that all of these examples would have the commands on a single line. That way you could cut-and-paste the command and run it yourself. Having them broken across multiple lines means you can't easily do this. It's OK if it makes the line too long. We can add an exception in `.flake8`.

Reply:
Something like this?
```
per-file-ignores =
    # line too long
    tests/pds4_create_xml_index_blackbox.py: E501
```

Reply:
Yes